packages/framework/src/Form/Admin/Product/Brand/BrandFormType.php

```diff
@@ -80,7 +82,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ];
         }
 
-        $builder
+        $builderBasicInformationGroup = $builder->create('basicInformation', GroupType::class, [
+            'label' => t('Basic information'),
+        ]);
+        $builderBasicInformationGroup
+            ->add('id', DisplayOnlyType::class, [
+                'label' => t('ID'),
+                'data' => $brand !== null ? $brand->getId() : 0,
```

Comment:
Why are you showing ID 0 for new Brands? The input should not be displayed is such a case.